pylint.out

```diff
@@ -0,0 +1,188 @@
+./hs_collection_resource/views.py:125:25: E117 over-indented
+./hs_collection_resource/patches/add_hasPart_term_django_shell.py:20:1: F841 local variable 'ex' is assigned to but never used
+./hs_file_types/nc_functions/nc_utils.py:292:61: W504 line break after binary operator
+./hs_file_types/nc_functions/nc_utils.py:354:49: E225 missing whitespace around operator
```

Comment:
this pylint.out needs to be removed.

Reply:
@hyi It seems this file currently exists in develop. May be we have to add this file to gitignore. I am going to make this change part of this PR unless we create a separate issue. Your thoughts?

Reply:
@pkdash Sounds like a good idea to add this file in gitignore. Agreed it is a good idea to address this in this PR.